### DISCUSSION TITLE
RADIO framework

### DISCUSSION THREAD

**Discussion by avacado** [greatfrontend/greatfrontend#10] [Q&A] [answered]
Hi, 
I am new to greatFrontEnd and reading about **RADIO** framework to solve system design interview questions. Though I like the strategy for applying the framework in tacking the problem, I am not convinced that 40% of the time should be spent on **Optimizations and Deep Dive**. Would love to know your perspective.

**Comment by gfeeng** [comment, marked as answer]
Hi @avacado, thanks for the question! If you look at the amount of content of the optimizations section compared to the other sections, it is definitely the longest and usually even longer than all the other sections combined, hence our recommendation of spending around 40% of the session discussing the various optimization techniques.

However, the recommended duration is not a hard rule and the actual duration per section depends on the problem. For example for the e-commerce website, there are more APIs to cover, so we'd probably spend more time on API and data model.
